go/obscuronode/config/enclave_config.go

```diff
@@ -30,8 +30,11 @@ type EnclaveConfig struct {
 	WriteToLogs bool
 	// The path that the node's logs are written to
 	LogPath string
-	// Whether enclave should use in-memory or persistent storage
+	// Whether the enclave should use in-memory or persistent storage
 	UseInMemoryDB bool
+	// Whether the enclave should encrypt responses to sensitive requests with viewing keys
+	// TODO - Consider removing this option and forcing the simulations to generate viewing keys.
```

Comment:
it would be good to raise a ticket as well, so we don't forget about this

Reply:
Ok. So you're sure we want to do that? Have the viewing keys enabled during the tests?

Reply:
yeah, totally.

Reply:
Ok, will raise.

Reply:
https://github.com/obscuronet/obscuro-internal/issues/513